testsuite/features/build_validation/retail/sle11sp4_kiwi_image_buildhost.feature

```diff
@@ -21,7 +21,7 @@ Feature: Bootstrap a SLES 11 SP4 Salt build host via the GUI
 
   Scenario: Check the new bootstrapped SLES 11 SP4 build host in System Overview page
     Given I am authorized as "admin" with password "admin"
```

Comment:
Do we still need this here? Why not use the new scenario with the login?

Reply:
fixed.